test/algorithms/simulation/sidb/quicksim.cpp

```diff
@@ -43,14 +43,13 @@ TEMPLATE_TEST_CASE(
 
     const auto simulation_results = quicksim<TestType>(lyt, quicksim_params);
 
-    CHECK(simulation_results.charge_distributions.empty());
+    CHECK(simulation_results.charge_distributions.size() == 2);
```

Comment:
Especially in this case, it is super counterintuitive to have two entries even though an empty layout was simulated. I would highly recommend keeping it the way it was.